rclcpp/include/rclcpp/create_timer.hpp

```diff
@@ -92,7 +96,8 @@ create_wall_timer(
   CallbackT callback,
   rclcpp::CallbackGroup::SharedPtr group,
   node_interfaces::NodeBaseInterface * node_base,
-  node_interfaces::NodeTimersInterface * node_timers)
+  node_interfaces::NodeTimersInterface * node_timers,
+  bool autostart = true)
```

Comment:
the new `autostart` argument should added to the doxygen description of this function

Reply:
Sure thing!